rust/pkg/cardano_serialization_lib.js.flow

```diff
@@ -761,6 +775,12 @@ declare export class BigNum {
    */
   is_zero(): boolean;
 
+  /**
+   * @param {BigNum} other
+   * @returns {BigNum}
+   */
+  div(other: BigNum): BigNum;
```

Comment:
Is this div floored? If so - let's put it into the name please for 100% clarity: `.div_floor`